code/src/index.css

```diff
@@ -5,9 +5,177 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  font-weight: 500;
+  font-size: 16px;
+  background: #fff;
+  display: flex;
+  justify-content: center;
+  align-items: center;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+/* find it in app.js------------- */
+main {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  justify-content: space-between;
+  margin: 5px 20px 40px 50px;
+  padding: 10px 20px 20px 50px;
+  margin-top: 15px;
+  width: 510px;
+  height: 200px;
+  word-wrap: break-word;
+}
+
+/* Styling MessageInput -----------------------------------------------*/
+
+h1 {
+  margin-top: 10px;
+  font-size: 23px;
+  text-align: center;
+  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
+    monospace;
+}
+.new-message-container {
+  width: 500px;
+  height: 400px;
+  word-wrap: break-word;
+}
+
+form {
+  margin-top: 30px;
+  padding: 10px 10px 10px 10px;
+  border: 3px solid #4d4d4d;
+  background: #f2f0f0;
+  box-shadow: 6px 7px;
+}
+
+.input-text {
+  margin: 8px;
+  font-size: 20px;
+}
```

Comment:
A little tip I learned when styling the textarea is that if you want to remove the small handle on the bottom right hand side of the box you can add resize: none. You can also set the overflow: auto which allows for the box to have a scroll bar if the user writes a super long message in the box.

Reply:
👌🏽 nice tips!